frontend/ui/widget/inputdialog.lua

```diff
@@ -635,7 +640,7 @@ function InputDialog:onClose()
     self._top_line_num = self._input_widget.top_line_num
     self._charpos = self._input_widget.charpos
     if self.view_pos_callback then
-        -- Give back top line num and cursor position
+        -- Let the widget store the current top line num and cursor position
```

Comment:
"the widget" in all the comments in this commit feels a bit confusing. For me, a "widget" is some UI small element (ie. InputDialog, InfoMessage). I think it would at least need "upper" or "parent": "upper widget".
And widgets don't usually handle/store settings, so "application", "using code", "upper code", something like that would sound better.

Reply:
Agreed, and doubly so because the widget is actually, well, *this* widget ;D.